src/js/components/Diagram/doc.js

```diff
@@ -64,6 +98,13 @@ export const themeDoc = {
     type: 'string | {dark: string, light: string}',
     defaultValue: 'accent-1',
   },
+  'global.animation': {
+    description: 'The animation configuration for Diagram.',
+    type: 'string',
+    defaultValue: `{
+      duration: '1s'
+    }`,
```

Comment:
`type` is contradicting the `defaultValue`, is it a string or an object?

Reply:
Fixed this to be `type: object`